### PR TITLE
Fixed the formatting and added a step.

### PR DESCRIPTION
**Pull request by RussDev7**
Changes made to the [README.md](https://github.com/AlexFlipnote/GiveMeBadge/blob/master/README.md) file.
- Fixed the formatting on the file in some areas. 
- Improved the quality of life by adding hyper links in some areas.
- Added a new step to explain how to use the Discord developer panel and acquire the bot token.
- Further addressed risks associated by the token.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,36 +3,41 @@
...
+- Discord server that you own.
+  - `Settings` -> `Enable Community` (if you haven't already)
 
-## Video tutorial 📼
+## Video Tutorial 📼
 If you need visual help on how to do this, then [click here and watch this video](https://i.alexflipnote.dev/7TzozoL.mp4)
 
 ## Usage ✨
-1. Download and install Python if you haven't already
+1. Download and install [Python](https://discord.com/developers/applications) if you haven't already.
```
Comment (AlexFlipnote) [inline README.md:16]: Didn't know Discord developer URL was a Python download link lmao. Should maybe make it use the recommended URL. (3.10.2)

File: README.md [hunk 2]
```diff
@@ -3,36 +3,41 @@
 </p>
 
 # Requirements 🧾
-- Discord Bot (https://discord.com/developers/applications)
-- Python 3.6 or above (https://www.python.org/downloads)
+- Discord developer panel: (https://discord.com/developers/applications)
```
Comment (AlexFlipnote) [inline README.md:6]: I like the wording, however it still somehow has to say "You need a bot"

**Review by AlexFlipnote — CHANGES_REQUESTED** [review]
Would probably look in to these before going forward.

File: README.md (hunk 1 above)
Comment (RussDev7) [reply README.md:16]: How embarrassing lol. I will update this.

**Review by AlexFlipnote — APPROVED** [review]

**Pull request merged**